iothub/device/tests/Authentication/DeviceClientConnectionStringExceptionTests.cs

```diff
@@ -13,127 +13,127 @@ namespace Microsoft.Azure.Devices.Client.Test.ConnectionString
     public class DeviceClientConnectionStringExceptionTests
     {
         [TestMethod]
-        [ExpectedException(typeof(InvalidOperationException))]
         public async Task DeviceClientConnectionString_ExtraModuleId_ExceptionTest()
         {
-            string connectionString = "SharedAccessKeyName=AllAccessKey;HostName=acme.azure-devices.net;DeviceId=device1;ModuleId=module1;SharedAccessKey=dGVzdFN0cmluZzE=";
-            await using var deviceClient = new IotHubDeviceClient(connectionString);
+            const string connectionString = "SharedAccessKeyName=AllAccessKey;HostName=acme.azure-devices.net;DeviceId=device1;ModuleId=module1;SharedAccessKey=dGVzdFN0cmluZzE=";
```

Comment:
In [IotHubConnectionStringBuilderTests.cs], the parameter strings are defined and shared throughout all the tests. Should we do the same for all tests needing connection strings?